### DISCUSSION TITLE
Mindustry

### DISCUSSION THREAD

**Discussion by shunnydie** [community/community#132045] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Здравствуйте!
столкнулся с проблемой с заходом на сектор в Mindustry под названием <Центр исследования биомасс>
при заходе ИМЕННО на него вылетает ошибка <Could not skip bytes. Expected length; 65535; Actual length: 53>
подскажите что делать?

**Comment by LiteBrite82** [comment]
Hi @shunnydie , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
